tests/unit/s2n_cipher_suites_test.c

```diff
@@ -50,11 +65,9 @@ int main()
                 security_policy = security_policy_selection[policy_index].security_policy;
                 cipher_preferences = security_policy->cipher_preferences;
                 for (size_t cipher_index = 0; cipher_index < cipher_preferences->count; cipher_index++) {
-                    /* The null cipher suite is just a placeholder, and is not included */
-                    if (cipher_preferences->suites[cipher_index] == &s2n_null_cipher_suite) {
+                    if (is_placeholder_cipher(cipher_preferences->suites[cipher_index])) {
```

Comment:
Not exactly following this change. Why exactly would the equal preference ciphers be in any security policy at this point?

Reply:
Oh, this was added when I had the new security policies added. I can remove them now but I can also just keep them here since new security policies will be added after this pr.

Reply:
Ah I see. You can keep it, I don't mind.